test/e2e/framework/framework.go

```diff
@@ -55,6 +56,8 @@ type Framework struct {
 	Clientset_1_2 *release_1_2.Clientset
 	Clientset_1_3 *release_1_3.Clientset
 
+	FederationClientset *federation_internalclientset.Clientset
+	// TODO: remove FederationClient, all the client access must be through FederationClientset
```

Comment:
@madhusudancs is the plan to remove `FederationClient` and switch over the ginkgo code to use `FederationClientSet` in a subsequent PR? Is there anything blocking us from doing that now?